### PR TITLE
fix: add team support link

### PR DESCRIPTION
**Pull request by leangseu-edx**
**TL;DR -** add missing link to the anchor tag for team

JIRA: [AU-604](https://openedx.atlassian.net/browse/AU-604)

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

- Setup team and follow the instruction of AU-604
- the link should lead you to [team support document](https://support.edx.org/hc/en-us/articles/360000191067-Submit-your-response#h_01FVD8SXM9E5H2DNAG87X25ZHR)

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Review by matthugs — APPROVED** [review]
I'm concerned about translation, but this seems like a value-add

File: openassessment/templates/openassessmentblock/response/oa_response.html [hunk 1]
```diff
@@ -314,7 +314,7 @@ <h5 class="message__title">{% trans "We could not delete files" %}</h5>
                       {% trans "One team member should submit on behalf of the entire team."%}
                   {% endif %}
                   {% blocktrans %}
-                  Learn more about team assignments here: (<a href="" >link</a>)
+                  Learn more about team assignments here: (<a target="_blank" href="https://support.edx.org/hc/en-us/articles/360000191067-Submit-your-response#h_01FVD8SXM9E5H2DNAG87X25ZHR">link</a>)
```
Comment (jansenk) [inline openassessment/templates/openassessmentblock/response/oa_response.html:317]: this doesn't really have to be a blocktrans. We should just be doing `{% trans "Learn more about team assignments here:" %} ` but it's not that big of a deal and I think you'd have to re-run translations which is a pain... i'll leave it up to you

**Review by jansenk — APPROVED** [review]

File: openassessment/templates/openassessmentblock/response/oa_response.html (hunk 1 above)
Comment (leangseu-edx) [reply openassessment/templates/openassessmentblock/response/oa_response.html:317]: I think the previous person try to translate `link` as well. The translation doesn't change I ran and review it in the local change.
